src/sentry/static/sentry/app/utils/fetchSentryAppInstallations.tsx

```diff
@@ -0,0 +1,12 @@
+import SentryAppInstallationStore from 'app/stores/sentryAppInstallationsStore';
+import {Client} from 'app/api';
+import {SentryAppInstallation} from 'app/types';
+
+const fetchSentryAppInstallations = async (api: Client, orgSlug: string) => {
+  const installsUri = `/organizations/${orgSlug}/sentry-app-installations/`;
+
+  const installs: SentryAppInstallation[] = await api.requestPromise(installsUri);
+  SentryAppInstallationStore.load(installs);
+};
+
+export default fetchSentryAppInstallations;
```

Comment:
This file has been greatly simplified since we no longer need to fetch the sentry apps (we get the app name from the sentry-component API)